src/utils/pageWrappers/__tests__/withSentry.test.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import {
+  withSentry,
+  withSentrySSR,
+  topLevelCatchBoundary,
+} from 'src/utils/pageWrappers/withSentry'
+import * as Sentry from '@sentry/node'
+import { flowRight } from 'lodash/util'
+import { mount } from 'enzyme'
+import { useAuthUser } from 'next-firebase-auth'
+import getMockNextJSContext from 'src/utils/testHelpers/getMockNextJSContext'
+import getMockAuthUser from 'src/utils/testHelpers/getMockAuthUser'
+
+jest.mock('react-relay')
+jest.mock('src/utils/relayEnvironment')
+
+const getMockCtxWithAuthUser = () => ({
+  ...getMockNextJSContext(),
+  AuthUser: getMockAuthUser(),
+})
+jest.mock('next-firebase-auth', () => ({
+  useAuthUser: jest.fn(),
+}))
+jest.mock('@sentry/node', () => {
+  let scopedUser = ''
+  return {
+    setUser: (user) => {
```

Comment:
It would be cleaner to use a mock function here, like: `setUser: jest.fn()`. Then, in tests you can assert: `expect(Sentry.setUser).toHaveBeenCalledWith(something)`.

In this approach, don't forget to clear Jest mocks after each test.